infrastructure/lambda.tf

```
@@ -0,0 +1,72 @@
+module "lambda_tracking" {
+  source              = "git@github.com:trustedshops/mars-terraform-modules.git//aws-lambda-function/?ref=feature/tf12"
+  service_name        = var.service_name
+  function_name       = "datalake-sender"
+  description         = "send api tracking data to BI teams firehose"
+  filename            = data.archive_file.lambda_api_tracking_dummy_zip.output_path
+  with_extra_policy   = true
+  extra_policy        = data.aws_iam_policy_document.lambda_api_tracking.json
+  lambda_tracing_mode = var.lambda_tracing_mode
+  lambda_runtime      = "nodejs14.x"
+  lambda_handler      = "index.default"
+  memory_size         = 256
+  timeout             = 30
```

We should set reserved concurrency

```suggestion
  source                         = "git@github.com:trustedshops/mars-terraform-modules.git//aws-lambda-function/?ref=feature/tf12"
  service_name                   = var.service_name
  function_name                  = "datalake-sender"
  description                    = "send api tracking data to BI teams firehose"
  filename                       = data.archive_file.lambda_api_tracking_dummy_zip.output_path
  with_extra_policy              = true
  extra_policy                   = data.aws_iam_policy_document.lambda_api_tracking.json
  lambda_tracing_mode            = var.lambda_tracing_mode
  lambda_runtime                 = "nodejs14.x"
  lambda_handler                 = "index.default"
  memory_size                    = 256
  timeout                        = 30
  reserved_concurrent_executions = 20
```